hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/utils/forensics/RecordParsers.java

```diff
@@ -35,6 +43,11 @@
  * runtime.
  */
 public class RecordParsers {
+    // A token that we can use to distinguish sidecar files with names like,
+    //   2022-12-05T14_23_46.192841556Z_01.rcd.gz
+    // from record stream files whose names do *not* include a _XX index
+    // like _01, _02, etc.
+    private static final String SIDECAR_ONLY_TOKEN = "Z_";
```

Comment:
How about something closer to 'sidecar' like `"SCAR_"` or `"SC_"` (Not gonna lie, kinda like SCAR)

Reply:
Ah I get your point...something like,
```
2022-12-05T14_23_46.192841556Z_SCAR_01.rcd.gz
```
yes that would have been nice...I think we have a contract with mirror nodes with this format now tho. 🤷